data-serving/data-service/src/model/case.ts

```diff
@@ -259,34 +288,34 @@ interface Source {
 interface Symptoms {
     provided: [string];
     imputed: [string];
-    other: [string];
 }
 
 interface DateRange {
     start: Date;
     end: Date;
 }
 
+interface TravelDates {
```

Comment:
Same comment as above re: interface necessity.

Reply:
see convo above -- whatever we decide for Event.date, I'll apply here too